app/code/Magento/Catalog/Test/Mftf/Section/StorefrontProductPageSection.xml

```diff
@@ -25,5 +25,6 @@
         <element name="customOptionDropDown" type="select" selector="//*[@id='product-options-wrapper']//select[contains(@class, 'product-custom-option admin__control-select')]"/>
         <element name="qtyInputWithProduct" type="input" selector="//tr//strong[contains(.,'{{productName}}')]/../../td[@class='col qty']//input" parameterized="true"/>
         <element name="customOptionRadio" type="input" selector="//span[contains(text(),'{{customOption}}')]/../../input" parameterized="true"/>
+        <element name="onlyProductsLeft" type="block" selector="//div[@class='availability only']"/>
```

Comment:
Please, make the selector of this element more sophisticated. Like, include some parent classes as well. There's a chance of a second element on the page with "availability only" class.

Thank you.

Reply:
**Done**